setup.py

```diff
@@ -6,12 +6,7 @@
       author='Dave Hunt',
       author_email='dhunt@mozilla.com',
       url='https://github.com/mozilla/pytest-mozwebqa',
-      packages=['pytest_mozwebqa'],
-      py_modules=[
```

Comment:
I see you removed the `py_modules` key. What was the purpose of that being there before?